README.md

```diff
@@ -36,14 +36,16 @@ _Feel free to try this out using [our example APK](https://github.com/bamlab/and
 
 1. Install the profiler `yarn add --dev @bam.tech/appium-helper @perf-profiler/e2e`
 
-2. Create a TS script including a performance test, for instance `start.ts`
+2. Create a TS script including an e2e performance test,
```

Comment:
Je trouve ça bizarre de dire en 1. "Installer @bam.tech/appium-helper" et en 2. "Tu peux utiliser ce que tu veux"

Peut être bouger le yarn add --dev @bam.tech/appium-helper dans le 2. vu que c'est ici qu'on en parle ? Comme ça dans le 1. on n'installe vraiment que le profiler

Reply:
Ah en plus c'est déjà dans `packages/appium-helper/README.md`, raison de plus de l'enlever d'ici 😄

Reply:
Good point, changing this!